packages/cart/src/models/line-item.ts

```diff
@@ -94,13 +94,13 @@ export default class LineItem {
   @Property({ columnType: "jsonb", nullable: true })
   variant_option_values?: Record<string, unknown> | null
 
-  @Property({ columnType: "boolean", default: true })
+  @Property({ columnType: "boolean" })
   requires_shipping = true
```

Comment:
question: Now that you removed the default, it means that if we have an sql query to run we won't have the default value. was it intended? I think it is find to have both since they are acting at different levels

Reply:
Yes, the change was intended and per the recommendation of MikroORM docs. That said, I think you raise a good point, so good to have in mind. Since this will need changing more places, I would like to keep this for a separate PR, such that we don't address it now and forget to follow up.